### PR TITLE
OP-507 update the Setting section of the doc

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-507.  Updated the text and various screenshots for the Setting section of the User doc.

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -2263,349 +2263,342 @@ Once you have pressed the *[.underline]##T##ypes* button on the *_Settings_* men
...
-* *[.underline]##N##ew*: to enter a new type (cannot use an already existing code)
-* *[.underline]##E##dit*: to edit an already existing type (the code must be unchanged)
-* *[.underline]##D##elete*: to delete an already existing type (cannot be deleted if still used somewhere in the application)
+* *[.underline]##N##ew*: to enter a new type (cannot use an existing code)
+* *[.underline]##E##dit*: to edit an existing type (the code cannot be changed)
+* *[.underline]##D##elete*: to delete an existing type (cannot be deleted if still used somewhere in the application)
 * *[.underline]##C##lose*: to close the window
 
-The Disease Types defined in this functionality will be reflected in the entire application, like in Disease definition or in OPD (see
+The Disease Types defined in this functionality are used throughout the application.  For example, in definition of Diseases or in OPD (see
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: double space here

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -2621,32 +2614,32 @@ To insert a new Ward, you must provide the following required inputs:
 NOTE: Using this function one can define wards with 0 (zero) beds but with a pharmacy.
 This allows for "logical” departments (rather than physical) to manage pharmacy movements.
 
-The Wards defined in this functionality will be reflected in the entire application, like in Admission module, Pharmaceuticals Stock and Pharmaceuticals Stock Ward (see
+The Wards defined in this functionality are used throughout the entire application.  See for example, the Admission module, Pharmaceuticals Stock and Pharmaceuticals Stock Ward (see
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: double space here

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

File: doc_user/UserManual.adoc (hunk 2 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: see above

File: doc_user/UserManual.adoc (hunk 1 above)
Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: It doesn't make any difference; the renderer removes extra spaces:

image

Comment (mwithi) [reply doc_user/UserManual.adoc]: it makes difference in size? 😁

Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: Changed.  You missed the other 8 occurrences of multiple spaces at the end and the start of the next sentence.

Comment (mwithi) [reply doc_user/UserManual.adoc]: Thanks!

**Pull request merged**